common/src/main/java/com/wynntils/screens/partymanagement/PartyManagementScreen.java

```diff
@@ -413,7 +414,7 @@ private void inviteFromField() {
 
     private void refreshAll() {
         Models.Party.requestData();
-        Models.Friends.requestData();
+        Managers.TickScheduler.scheduleLater(Models.Friends::requestData, 5);
```

Comment:
Why?

Reply:
This was just testing to see if it was a ratelimit problem, removed now